### DISCUSSION TITLE
Can we configure the output from the interactive window to use double quotes instead of single quotes?

### DISCUSSION THREAD

**Discussion by leobeeson** [microsoft/vscode-jupyter#12173] [Questions and Answers]
I often copy output data (from a Python dict object) in the interactive window and analyse it on a text editor as "pretified" json data. This helps me quickly eyeball my development.

However, the interactive output is always printed with single quotes, so if I want to copy and paste it in a text editor and "prettify" the json to then analyse it, I first have to manually replace all single quotes with double quotes. This is not always trivial, specially with keys that have a single quote in the form of an apostrophe or a contraction (e.g. `I'm`).

I've tried looking for a settings parameter to do this, but haven't found anything for the interactive window. Is it possible? Or is the interactive output hardcoded to always print single quotes? 

I can always json.dump() to file, but that defeats the purpose of the interactive window, takes longer (and I might want to iterate an eyeball the output of some process several dozen times in one sitting), as well as forces me to persist data that I have no need to persist.

Similar discussions:
* [Customize "Python Interactive" Window in VS Code](https://stackoverflow.com/questions/61703209/customize-python-interactive-window-in-vs-code)

Thanks.

**Comment by amunger** [comment]
that's not supported at the moment, we just output whatever the kernel provides. It looks like the kernel has its own handling of when to use which quote, so would be tough for us to handle the different cases:

![image](https://user-images.githubusercontent.com/2019016/207678928-5e5a1e9e-80a2-496f-932a-a13e4eaeb97f.png)
